src/rpc/blockchain.cpp

```diff
@@ -2293,15 +2297,20 @@ UniValue scantxoutset(const JSONRPCRequest& request)
         g_scan_progress = 0;
         int64_t count = 0;
         std::unique_ptr<CCoinsViewCursor> pcursor;
+        CBlockIndex* tip;
         {
             LOCK(cs_main);
             ::ChainstateActive().ForceFlushStateToDisk();
             pcursor = std::unique_ptr<CCoinsViewCursor>(pcoinsdbview->Cursor());
             assert(pcursor);
+            tip = ::ChainActive().Tip();
```

Comment:
I'd just declare and initialize tip here instead of at the top and same with pcursor